--- libdbus-java-1.11/Makefile.orig	2006-11-14 17:24:46.000000000 +0100
+++ libdbus-java-1.11/Makefile	2006-11-16 18:24:39.636176750 +0100
@@ -15,8 +15,8 @@
 JAR?=jar
 INCLUDES?=`pkg-config --cflags dbus-1` -I${JAVA_HOME}/include -I${JAVA_HOME}/include/linux
 CFLAGS?= -Os -Wall -Werror 
-CFLAGS+=$(INCLUDES)
-CFLAGS+=$(call cc-option,-fno-stack-protector,)
+override CFLAGS+=$(INCLUDES)
+override CFLAGS+=$(call cc-option,-fno-stack-protector,)
 LIBS?=`pkg-config --libs dbus-1`
 CC?=gcc
 LD?=ld
@@ -94,7 +94,6 @@
 
 libdbus-java.so: dbus-java.o
 	$(LD) $(LDFLAGS) -fpic -shared -o $@ $^ $(LIBS)
-	$(STRIP) $@
 libdbus-java-$(VERSION).jar: .classes
 	(cd classes; $(JAR) -cf ../$@ org/freedesktop/dbus/*.class org/freedesktop/*.class org/freedesktop/Hal/*.class)
 dbus-java-test-$(VERSION).jar: .testclasses
